tests/test_repocard.py

```diff
@@ -97,6 +109,8 @@
     os.path.dirname(os.path.abspath(__file__)), "fixtures/repocard"
 )
 
+REPO_NAME = "dummy-hf-hub-{}".format(int(time.time() * 10e3))
```

Comment:
Probably better to use uuid here:

```suggestion
REPO_NAME = "dummy-hf-hub-{str(uuid.uuid4())}"
```